### PR TITLE
Created selection-flag start and end flag

### PR DESCRIPTION
**Pull request by tidoni**
I use it for selecting the starttime and endtime in an audio file.

### CODE REVIEW

**Comment by ericfennis** [comment]
Nice icon!

**Review by ericfennis — CHANGES_REQUESTED** [review]
Seems like there is a random square in the svg.
The rect element can be removed.
Can you also add a new line add the end of the files.

File: icons/selection-flag-end.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="100" height="100" x="50" y="50" />
+  <path d="M17 22 17 2 7 7 17 12"></path>
+</svg>
```
Comment (ericfennis) [inline icons/selection-flag-end.svg:14]:
```suggestion
  <path d="M17 22 17 2 7 7 17 12" />
</svg>

```

File: icons/selection-flag-start.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="100" height="100" x="50" y="50" />
+  <path d="M7 22 7 2 17 7 7 12"></path>
+</svg>
```
Comment (ericfennis) [inline icons/selection-flag-start.svg:14]:
```suggestion
  <path d="M7 22 7 2 17 7 7 12" />
</svg>

```

**Review by locness3 — CHANGES_REQUESTED** [review]
What about `triangle-flag-{left,right}`?

**Comment by ericfennis** [comment]
I agree with @locness3, with generalizing the name. We make it more multy-purpose. Fits well with our other names.
I like `flag-triangle-{left,right}` more, because `flag` is the icon and `triangle` is the variation.

**Pull request merged**